cheetah/utils.py

```diff
@@ -9,3 +12,47 @@ def __call__(self):
         name = f"{self._prefix}_{self._counter}"
         self._counter += 1
         return name
+
+
+def generate_3d_uniform_ellipsoid(
+    num_particles: torch.Tensor, r_x: torch.Tensor, r_y: torch.Tensor, r_z: torch.Tensor
```

Comment:
I think the arguments should be called `radius_x` and so on. It's not too long and this way I don't have to look at the documentation to figure out what `r_x` is ... i.e. it's much more readable.